### PR TITLE
[aoc-collector non-root user]:  Statement breaking change non root user

### PR DESCRIPTION
**Pull request by PaurushGarg**
**Description:** <Describe what has changed.>

PR to add log statement regarding breaking change for users of `awscontainerinsightreceiver` component due to upcoming non-root user image release of ADOT Collector.
Dependent on PR: https://github.com/aws-observability/aws-otel-collector/pull/2297

**Link to tracking Issue:** <Issue number if applicable>
https://github.com/aws-observability/aws-otel-collector/issues/2222

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Use the community resources below for getting help with the ADOT Collector.
 ### Notice: ADOT Collector v0.33.0 Breaking Change
 Users of the `statsd` receiver, please refer to GitHub Issue - [Warning: StatsD Receiver → EMF Exporter Metric Pipeline Breaking Change](https://github.com/aws-observability/aws-otel-collector/issues/2249)
 for information on an upcoming breaking change.
+Users of the 'awscontainerinsightreceiver' receiver, please refer to the GitHub Issue - [Warning: AWSContainInsightReceiver → Non-Root User Breaking Change](https://github.com/aws-observability/aws-otel-collector/issues/****) for more information on the upcoming breaking change.
```
Comment (bryan-aguilar) [inline README.md:24]: This should be split into it's own header. Follow the example above for the statsd breaking change.

File: cmd/awscollector/main.go [hunk 2]
```diff
@@ -97,6 +97,10 @@ func buildAndParseFlagSet(featgate *featuregate.Registry) (*flag.FlagSet, error)
 	log.Printf("attn: users of the statsd receiver please refer to " +
 		"https://github.com/aws-observability/aws-otel-collector/issues/2249 in regards to an ADOT Collector v0.33.0 " +
 		"breaking change")
+	// TODO: remove after ADOT Collector v0.35.0 is released
+	log.Printf("attn: users of the awscontainerinsightreceiver please refer to " +
+		"https://github.com/aws-observability/aws-otel-collector/issues/**** in regards to an ADOT Collector v0.33.0 " +
```
Comment (bryan-aguilar) [inline cmd/awscollector/main.go:102]: What version will this breaking change actually happen?

File: README.md (hunk 1 above)
Comment (PaurushGarg) [reply README.md]: Thanks. Updated.

File: cmd/awscollector/main.go (hunk 2 above)
Comment (PaurushGarg) [reply cmd/awscollector/main.go]: Thanks. Updated.

File: cmd/awscollector/main.go [hunk 3]
```diff
@@ -97,6 +97,10 @@ func buildAndParseFlagSet(featgate *featuregate.Registry) (*flag.FlagSet, error)
 	log.Printf("attn: users of the statsd receiver please refer to " +
 		"https://github.com/aws-observability/aws-otel-collector/issues/2249 in regards to an ADOT Collector v0.33.0 " +
 		"breaking change")
+	// TODO: remove after ADOT Collector v0.35.0 is released
+	log.Printf("attn: users of the awscontainerinsightreceiver please refer to " +
+		"https://github.com/aws-observability/aws-otel-collector/issues/**** in regards to an ADOT Collector v0.35.0 " +
```
Comment (bryan-aguilar) [inline cmd/awscollector/main.go:102]: update to link with actual issue when created

**Review by bryan-aguilar — CHANGES_REQUESTED** [review]

File: cmd/awscollector/main.go (hunk 3 above)
Comment (PaurushGarg) [reply cmd/awscollector/main.go]: Updated.

File: README.md [hunk 4]
```diff
@@ -22,6 +22,9 @@ Use the community resources below for getting help with the ADOT Collector.
 Users of the `statsd` receiver, please refer to GitHub Issue - [Warning: StatsD Receiver → EMF Exporter Metric Pipeline Breaking Change](https://github.com/aws-observability/aws-otel-collector/issues/2249)
 for information on an upcoming breaking change.
 
+### Notice: ADOT Collector v0.35.0 Breaking Change
+Users of the 'awscontainerinsightreceiver' receiver, please refer to the GitHub Issue - [Warning: Container Image Default User Change → Important consideration for AWSContainerInsightReceiver](https://github.com/aws-observability/aws-otel-collector/issues/2317) for more information on the upcoming breaking change.
```
Comment (bryan-aguilar) [inline README.md:26]:
```suggestion
Users of the 'awscontainerinsightreceiver', please refer to the GitHub Issue - [Warning: Container Image Default User Change → Important consideration for AWSContainerInsightReceiver](https://github.com/aws-observability/aws-otel-collector/issues/2317) for more information on an upcoming breaking change.
```

**Review by bryan-aguilar — APPROVED** [review]

**Pull request merged**